OpenSim/Tools/Test/testVisualization.cpp

```diff
@@ -476,7 +476,7 @@ void populate_wrapModelPrimitives(SimTK::Array_<DecorativeGeometry>& stdPrimitiv
     // that DecorativeCylinder is Y aligned while WrapCylinder is Z aligned
     cylTransform.updR().setRotationFromAngleAboutX(SimTK_PI / 2);
     stdPrimitives.push_back(
-        DecorativeCylinder(.025, .05).setBodyId(0).setColor(SimTK::Cyan)
+        DecorativeCylinder(.025, .05).setBodyId(0).setColor(Vec3(0, 0.2, 0.8))
```

Comment:
Cyan is `[0 1 1]` and the model file here has `[0 0.2 0.8]`. Since the color from the model file was not being processed previously in updateFromXMLNode(), we lost the non-default color of `[0 0.2 0.8]`.